installers/windows.ps1

```diff
@@ -0,0 +1,105 @@
+<#
+.Parameter Uninstall
+    Whether to uninstall an existing native messenger installation.
+.Parameter InstallDirectory
+    The directory in which the native messenger should be installed. Defaults
+    to "$env:USERPROFILE\.tridactyl".
+.Parameter Tag
+    The Tridactyl version to which the corresponding version of the native
+    messenger should be installed. Tridactyl versions lower than 1.21.0 are not
+    supported.
+#>
+Param (
+    [switch]$Uninstall = $false,
+    [string]$InstallDirectory = "$env:USERPROFILE\.tridactyl",
+    [string]$Tag = "1.21.0"
```

Comment:
```suggestion
    [string]$Tag = "master"
```

This way beta builds that don't really have a concept of a supported version of the native messenger can always be on the most up-to-date one.